modules/websocket/lws_client.cpp

```diff
@@ -34,7 +34,10 @@
 #include "core/io/ip.h"
 #include "core/io/stream_peer_ssl.h"
 #include "core/project_settings.h"
-#include "tls/mbedtls/wrapper/include/openssl/ssl.h"
+#if defined(LWS_OPENSSL_SUPPORT)
+// Not openssl, just the mbedtls wrapper
+#include "openssl/ssl.h"
```

Comment:
Isn't that a bit risky given that many users will have `/usr/include/openssl/ssl.h` in their system which could take precedence here?

Reply:
Yes, it is, which is why I hate this library, it's simply ill-conceived in so many aspects.
The issue has been there for a while, I worked it around using `Prepend` instead of `Append` in 659c62ee338975eddf1f7e65f68190af65199324 / #17115